style.css

```diff
@@ -0,0 +1,443 @@
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  position: absolute;
+  top: 14px;
+  left: 10px;
+}
+
+.Union {
+  position: absolute;
+  /* stylelint-disable-next-line unit-no-unknown */
+  top: 14px;
+  right: 10px;
+}
+
+.social-icons {
+  list-style: none;
+  padding: 0;
+  display: flex;
+  width: 170.17px;
+  height: 20px;
+  order: 3;
+  flex-grow: 0;
+  margin-left: 0.2rem;
+}
+
+.social-icons li {
+  margin-right: 10px;
+}
+
+.social-icons li:last-child {
+  margin-right: 0;
+}
+
+/* stylelint-disable-next-line block-opening-brace-space-before */
+#my-section {
+  /* stylelint-disable-next-line declaration-colon-space-after */
+  background-image: url(img/header-shapes\ mobile@2x.svg);
+  background-size: cover;
+  background-position: center;
+  width: 381px;
+}
+
+header {
+  margin-top: -3rem;
+}
+
+.container h1 {
+  margin-top: 2rem;
+}
+
+.together {
+  margin-bottom: 10rem;
+}
+
+/* stylelint-disable-next-line no-descending-specificity */
+p {
+  font-family: "Poppins", sans-serif;
+}
+
+.together p {
+  width: 295px;
+  height: 76px;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+h3 {
+  font-family: "Poppins", sans-serif;
+}
+
+ul {
+  margin-left: -2.6rem;
+}
+
+.my-button {
+  display: inline-block;
+  font-family: "Poppins", sans-serif;
+  flex-direction: column-reverse;
+  padding: 4px 12px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  color: #6070ff;
+  border: none;
+  margin-top: -13rem;
+}
+
+.js {
+  width: auto;
+}
+
+.this-button {
+  display: inline-block;
+  flex-direction: column-reverse;
+  padding: 4px 12px;
+  width: 92px;
+  height: 24px;
+  background: #ebebff;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  color: #6070ff;
+  border: none;
+  margin-bottom: -3rem;
+  /* stylelint-disable-next-line property-no-unknown */
+  font-size: 12px;
+}
+
+.see-project {
+  height: 42px;
+  /* stylelint-disable-next-line font-family-no-missing-generic-family-keyword */
+  font-family: "Poppins";
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  /* stylelint-disable-next-line color-hex-case */
+  color: #396df2;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  border: 1px solid;
+  /* stylelint-disable-next-line color-hex-length */
+  background: #ffffff;
+  border-radius: 8px;
+  margin-bottom: 4rem;
```

Comment:
- [ ] According to your previouse reviewer kindly Please make sure you added the hover and active properties to your button. Kindly check the Figma design.
- [ ] ![Screenshot 2023-06-04 000218](https://github.com/Edgahkipkemoi/mobile/assets/92458236/e909995b-4946-403c-8285-ab721d671e93)
- [ ] see screenrecorf  

https://github.com/Edgahkipkemoi/mobile/assets/92458236/87f3ff1c-087a-4849-b717-a889596b1628